integration/mochaTests/README.md

```diff
@@ -0,0 +1,131 @@
+# Mocha Tests
+
+Currently, integration tests are being transitioned over from KITE to Mocha. `mochaTests` contains the tests that are being transitioned from KITE to Mocha.
+
+## Test Types
+
+There are two types of Mocha tests: integration and browser compatibility.
+
+### Integration tests
+Integration tests are minimal tests that run on latest Chrome on MacOs. These tests are used to test the basic functionality across the more popular browsers.
+
+### Browser Compatibility Tests
+Browser compatibility tests are used to test a feature across several different browser and OS combinations. This test is used to make sure that JS SDK features are compatible with a wide variety of browser and OS. There is a default set of browser and OS that are used for this test. In a case where a feature is not compatible with one of the browser or OS then the user can pass in a custom config.
+
+In the following section, we will go over the schema of the custom JSON config.
+#### JSON Config Schema
+
+```json
+{
+   "clients": [
+     {
+        /* 
+          Browser name
+          @required
+        */ 
+        "browserName": "chrome",
+        /* 
+          Browser Version
+          @required
+        */
+        "browserVersion": "latest",
+        /* 
+          Platform
+          @required        
+        */
+        "platform": "MAC"
+     }
+   ]
+}
+```
+## Running tests
+
+### Running Integration Test Locally
+You can run the integration tests locally. By default, integration tests will run on latest version of Chrome that is installed on your machine.
+You will have to make sure that you have the required drivers installed on your machine that the selenium test uses. You can find more information about the different drivers available at the `selenium-webdriver` npm page: [https://www.npmjs.com/package/selenium-webdriver](https://www.npmjs.com/package/selenium-webdriver). 
+
+If you have an older version of driver installed then you will need to have an older version of browser on your machine as well. Generally, it is recommended to do local testing on latest version. If you need to test on older version, you can run a browser compatibility test with saucelabs.
```

Comment:
- `installed then` => `installed, then` (add a comma)
- `on latest` => `on the latest`
- `test on older version` => `test on an older version`
- `saucelabs` => `Sauce Labs`